addons/auth_totp/controllers/home.py

```diff
@@ -7,7 +7,7 @@
 from odoo.addons.web.controllers import home as web_home
 
 TRUSTED_DEVICE_COOKIE = 'td_id'
-TRUSTED_DEVICE_AGE = 90*86400 # 90 days expiration
+TRUSTED_DEVICE_AGE = 90  # 90 days expiration
```

Comment:
if you are going to change the meaning of a global variable, it’s best to change its name. This way, you are sure nobody will use a value 86.000 smaller than expected.